### PR TITLE
Update KNP

### PR DESCRIPTION
**Pull request by sbathgate**
- Rectified the knp playbook README
- Removed documize >> conversion knp
- Removed obsolete nsp

### CODE REVIEW

File: openshift/templates/knp.yaml [hunk 1]
```diff
@@ -95,6 +95,52 @@ objects:
...
+      podSelector:
+        matchLabels:
+          deploymentconfig: ${NAME}${SUFFIX}
+      ingress:
+        - from:
+            - podSelector:
+                matchLabels:
+                  deploymentconfig: ${NAME}-conversion${SUFFIX}
+          ports:
+            - protocol: http
```
Comment (ShellyXueHan) [inline openshift/templates/knp.yaml]: When i apply them, got error message. It doesn't look like `http` and `https` are supported protocol. If you run `oc explain NetworkPolicy.spec.ingress.ports.protocol` only TCP, UDP, or SCTP are allowed.

**Comment by yhfreeman** [comment]
Removed documize >> conversion knp because we ask documize to use the converison by giving documize the conversion url and all calls to the conversion are made from the external.

**Comment by yhfreeman** [comment]
@ShellyXueHan [Test documize](https://documize-test-101ed4-test.apps.silver.devops.gov.bc.ca/) seems fine after the knp playbook was run from this branch.
The playbook in theory should work in prod, assuming the documize pods are labelled with `deploymentconfig=documize-prod`. Many thanks.

**Review by ShellyXueHan — APPROVED** [review]
dope!

**Pull request merged**